client/components/tinymce/plugins/insert-menu/style.scss

```diff
@@ -0,0 +1,31 @@
+.mce-container .mce-wpcom-insert-menu {
+	& .mce-caret {
+		padding-left: 12px;
+	}
+}
+
+.mce-container.mce-menu .mce-container-body .mce-wpcom-insert-menu__menu-item {
+	margin-left: -6px;
+
+	& svg {
```

Comment:
Also, prefixing descendent selector with `&` is redundant and can be removed safely.